run-ab-platform.sh

```diff
@@ -39,13 +40,110 @@ Help()
 docker_compose_debug_yaml="docker-compose.debug.yaml"
                   dot_env=".env"
               dot_env_dev=".env.dev"
-                     flags="flags.yml"
-             temporal_yaml="temporal/dynamicconfig/development.yaml"
+                    flags="flags.yml"
+            temporal_yaml="temporal/dynamicconfig/development.yaml"
 # any string is an array to POSIX shell. Space seperates values
 all_files="$docker_compose_yaml $docker_compose_debug_yaml $dot_env $dot_env_dev $flags $temporal_yaml"
 
 base_github_url="https://raw.githubusercontent.com/airbytehq/airbyte-platform/v$VERSION/"
 
+telemetrySuccess=false
+telemetrySessionULID=""
+telemetryUserULID=""
+telemetryEnabled=true
+# telemetry requires curl to be installed
+if ! command -v curl > /dev/null; then
+  telemetryEnabled=false
+fi
+
+# TelemetryConfig configures the telemetry variables and will disable telemetry if it cannot be configured.
+TelemetryConfig()
+{
+  # only attempt to do anything if telemetry is not disabled
+  if $telemetryEnabled; then
+    telemetrySessionULID=$(curl -s https://ulid.truestamp.com/ | grep ulid | cut -d ":" -f2 | xargs)
```

Comment:
I wonder if there's a local implementation we can access...

https://github.com/ulid/spec?tab=readme-ov-file#implementations-in-other-languages

Reply:
rolled out http://ulid.abapp.cloud/